test/acceptance/features/bindAppToService.feature

```diff
@@ -217,3 +217,55 @@ Feature: Bind an application to a service
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-empty-app" should be changed to "False"
         And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding Request "binding-request-empty-app" should be changed to "EmptyApplicationSelector"
 
+
+    Scenario: Backend Service status update gets propagated to the binding secret
+        Given OLM Operator Backend is running
+        * Backend CR "backend-demo" is applied
+            """
+            apiVersion: "stable.example.com/v1"
+            kind: Backend
+            metadata:
+                name: backend-demo
+                annotations:
+                    servicebindingoperator.redhat.io/status.ready: 'binding:env:attribute'
+                    servicebindingoperator.redhat.io/spec.host: 'binding:env:attribute'
+            spec:
+                host: example.common
+            """
+        * Service Binding request is applied
+            """
+            apiVersion: apps.openshift.io/v1alpha1
+            kind: ServiceBindingRequest
+            metadata:
+                name: binding-request-backend
+            spec:
+                backingServiceSelector:
+                    group: stable.example.com
+                    version: v1
+                    kind: Backend
+                    resourceRef: backend-demo
+                    id: SBR
+                customEnvVar:
+                  - name: CustomReady
+                    value: '{{ .SBR.status.ready }}'
+                  - name: CustomHost
+                    value: '{{ .SBR.spec.host }}'
+            """
+        Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding Request "binding-request-backend" should be changed to "True"
+        And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-backend" should be changed to "False"
+        Then Secret "binding-request-backend" contains CustomReady custom env name with value "<no value>"  
```

Comment:
That rephrase would be wrong. The secret indeed contains a CustomReady key because it is being mentioned in SBR. It does not rather contain the value. the value is `<no value>` . So basically what we will get is `CustomReady: <no value>` until we provide suitable annotations for exporting values from the service. The key would be intact. IMO we should not rephrase it

Reply:
Thanks for clarification. In that case, it would be better to rephrase it to:

```
 CustomReady key  in secret "binding-request-backend" has no value
```

Reply:
Done https://github.com/redhat-developer/service-binding-operator/pull/597/commits/b904a2186b93b06ef06213f2a23139572628abdc